README.md

```diff
@@ -2,56 +2,42 @@
 
 # Ruby LSP
 
-This gem is an implementation of the [language server protocol specification](https://microsoft.github.io/language-server-protocol/) for Ruby, used to improve editor features.
+The Ruby LSP is an implementation of the [language server protocol](https://microsoft.github.io/language-server-protocol/)
+for Ruby, used to improve rich features in editors. It is a part of a wider goal to provide a state-of-the-art
+experience to Ruby developers using modern standards for cross-editor features, documentation and debugging.
 
-# Overview
+Want to discuss Ruby developer experience? Consider joining the public
+[Ruby DX Slack workspace](https://join.slack.com/t/ruby-dx/shared_invite/zt-1s6f4y15t-v9jedZ9YUPQLM91TEJ4Gew).
 
-The intention of Ruby LSP is to provide a fast, robust and feature-rich coding environment for Ruby developers.
-
-It's part of a [wider Shopify goal](https://github.com/Shopify/vscode-shopify-ruby) to provide a state-of-the-art experience to Ruby developers using modern standards for cross-editor features, documentation and debugging.
-
-It provides many features, including:
-
-* Syntax highlighting
-* Linting and formatting
-* Code folding
-* Selection ranges
-
-It does not perform typechecking, so its features are implemented on a best-effort basis, aiming to be as accurate as possible.
-
-Planned future features include:
-
-* Auto-completion and navigation ("Go To Definition") ([prototype](https://github.com/Shopify/ruby-lsp/pull/429))
-* Support for plug-ins to extend behavior
-
-The Ruby LSP does not perform any type-checking or provide any type-related assistance, but it can be used alongside [Sorbet](https://github.com/sorbet/sorbet)'s LSP server.
-
-At the time of writing, these are the major differences between Ruby LSP and [Solargraph](https://solargraph.org/):
-
-* Solargraph [uses](https://solargraph.org/guides/yard) YARD documentation to gather information about your project and its gem dependencies. This provides functionality such as context-aware auto-completion and navigation ("Go To Definition")
-* Solargraph can be used as a globally installed gem, but Ruby LSP must be added to the Gemfile or gemspec if using RuboCop. (There are pros and cons to each approach)
-
-## Learn More
+## Usage
 
-* [RubyConf 2022: Improving the development experience with language servers](https://www.youtube.com/watch?v=kEfXPTm1aCI) ([Vinicius Stock](https://github.com/vinistock))
+If using VS Code, install the [Ruby LSP extension](https://github.com/Shopify/vscode-ruby-lsp) to get the extra features
```

Comment:
I think we need to be extra explicit about NOT to add it to Gemfile while using VS Code. I missed 
`No other configuration is required and you can skip this section.` part on the first pass so I assume some readers may do that too.

Example:

```
## Usage

### With VS Code

Just install the Ruby LSP extension and there's no need to install this gem manually.

### With other editors

See the editors doc....
```